azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/GroupCallOptions.java

```diff
@@ -3,56 +3,43 @@
 
 package com.azure.android.communication.ui;
 
-import android.content.Context;
 import com.azure.android.communication.common.CommunicationTokenCredential;
 import java.util.UUID;
 
 /**
  * Options to start group call experience using {@link CallComposite}.
  */
 public final class GroupCallOptions {
-    private final Context context;
```

Comment:
Removed from these classes because it's not really an "Option" to the GroupCall. We also can not guarantee where an API user will store this Options class, but it's a risk for a context leak.

E.g. Say it is put in a redux store, or a global history repository. While safe to store data, not really safe to store a stale context, it can leak memory and fail if re-used.